extensions/amp-list/0.1/amp-list.js

```diff
@@ -552,19 +554,16 @@ export class AmpList extends AMP.BaseElement {
       scheduleNextPass();
       current.rejecter();
     };
-    if (this.ssrTemplateHelper_.isSupported()) {
-      const html = /** @type {string} */ (current.data);
-      this.templates_.findAndSetHtmlForTemplate(this.element, html)
-          .then(result => this.updateBindings_([result]))
-          .then(element => this.render_(element, current.append))
-          .then(onFulfilledCallback, onRejectedCallback);
+    const isSSR = this.ssrTemplateHelper_.isSupported();
+    const renderTemplate = this.ssrTemplateHelper_.renderTemplate(
+        this.element, current.data)
+        .then(result => this.updateBindings_(isSSR ? [result] : result))
```

Comment:
Minor: A bit cleaner to change `updateBindings_` to automatically wrap its input in an array if it's not an array.

Reply:
done